Detectors/ITSMFT/ITS/simulation/include/ITSSimulation/Chip.h

```diff
@@ -146,6 +146,7 @@ class Chip : public TObject
     /// @param p Point to be added
     void InsertPoint(Point *p);
 
+    void SetGeometry(GeometryTGeo *g) { fGeometry=g; }
```

Comment:
@iouribelikov I propose that you and the original authors add a couple of tests for the functionality and a thorough description. Then somebody else could do the optimization and migration to O2 standards and policies. I belief we will not need many iterations  to train both sides of this process.